lmctl/environment/lmenv.py

```diff
@@ -87,6 +93,17 @@ def _validate_oauth(cls, values):
             raise ValueError(f'Secure TNCO environment cannot be configured with "api_key" when using "auth_mode={OAUTH_MODE}". Use "client_id/client_secret" or "username/password" combination or set "auth_mode" to "{ZEN_AUTH_MODE}". If the TNCO environment is not secure then set "secure" to False')
         return values
 
+    @classmethod
+    def _validate_okta(cls, values):
+        cls._validate_oauth(values)
```

Comment:
Errors thrown by `_validate_oauth` will mention oauth mode not okta mode, which will be confusing:

```
Secure TNCO environment must be configured with either "client_id" or "username" property when using "auth_mode=oauth}"
```

Might be best to write all validation logic specific to Okta in `_validate_okta` rather than rely on the existing oauth logic (even though it's similar)

Reply:
Addressed